dashboard/test/models/user_test.rb

```diff
@@ -696,27 +707,23 @@ class UserTest < ActiveSupport::TestCase
   end
 
   test 'sanitize_race_data sanitizes closed_dialog' do
-    user = create :student
-    user.update!(races: %w(white closed_dialog))
-    assert_equal %w(closed_dialog), user.reload.races
+    @student.update!(races: %w(white closed_dialog))
+    assert_equal %w(closed_dialog), @student.reload.races
   end
 
   test 'sanitize_race_data sanitizes too many races' do
-    user = create :student
-    user.update!(races: %w(white black hispanic asian american_indian hawaiian))
-    assert_equal %w(nonsense), user.reload.races
+    @student.reload.update!(races: %w(white black hispanic asian american_indian hawaiian))
```

Comment:
just curious if the `.reload` was added here for a necessary reason (no need to change since the tests are currently passing).